### DISCUSSION TITLE
合肥工业大学（翡翠湖校区）

### DISCUSSION THREAD

**Discussion by fautie** [CollegesChat/university-information#33] [University]
1. 宿舍是上床下桌吗
是的
2. 教室和宿舍有没有空调
教室有空调,可以用手机红外遥控器开启
3. 有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远
有独立卫浴(不通热水,需要自己打水加装自吸淋浴器(自己买))
有大澡堂,三楼,二楼和三楼使用不同楼梯,内部有单独隔间,但大厅常有人裸露生殖器走动.
4. 有早自习、晚自习吗
大一有早操有晚自习,大二晚上会检查宿舍有无人驻留.
5. 有晨跑吗
早操(约莫10分钟以上)
6. 每学期跑步打卡的要求是多少公里，可以骑车吗
It varies.
去年是打卡制度,今年是跑步凑整制度.用的自制小程序来记程.
小程序用的是腾讯的地图API,精度差.
学校目前在开发内网vpn系统,没开发完,如果开发完了据说会有单独的跑步打卡程序.
__据称__效果要和主流打卡软件趋同.
7. 寒暑假放多久，每年小学期有多长
严格按照每个学期教学20周的规定形式,寒暑假自己算.
据说本校是三学期制度但本人没体验过.
ps. 今年暑假只有一个月多一点
8. 学校允许点外卖吗，取外卖的地方离宿舍楼多远
无相关规定,也无专门的取外卖点.但学校不会阻止学生取外卖.
在封校期间和非封校期间学生都可以隔着铁栅栏拿外卖.
9. 学校附近有地铁站吗
有俩,一个在远离宿舍区和教学区的学校大门门口,一个在体育馆栅栏旁边.
宿舍区的大门附近有很多共享单车,地铁站口也有很多共享单车.
10. 宿舍楼有洗衣机吗
有
11. 校园网怎么样
限速只对IPv4,你懂的.
去年给所有宿舍升级了接入口到1Gbps.
通过技巧和勇气,努把力指不定你外网随便跑800Mbps
![image](https://user-images.githubusercontent.com/83117242/123650823-859ab980-d85d-11eb-8c1b-4ddf1e0213e8.png)

12. 每天断电断网吗，几点开始断
不断
13. 食堂价格贵吗，会吃出异物吗
这个校区有7个食堂,
南区有9个.
贵或不贵,全看你心情
至少我吃出来过铁丝
当然我十分庆幸不是吃出来其他东西
14. 洗澡热水供应时间
上午8点?还是9点?到晚上11点以后
15. 校园内可以骑电瓶车吗，电池在哪能充电
可以,可以,大概是某个车棚
16. 宿舍限电情况
限,但我不知道限制到哪里.
17. 通宵自习有去处吗
没那么卷,不知道
但宿舍可以通宵
如果觉得舍友恶心的话可以在宿舍大厅那里通宵
那里有个桌子
18. 大一能带电脑吗
可以
19. 学校里面用什么卡，饭堂怎样消费
校园卡可用,食堂只能刷校园卡.
超市可以刷校园卡也可以支付宝/weixin
20. 学校会给学生发银行卡吗
会发,20级是农行卡
不过奖学金/工资等需要一张中国银行储蓄卡,需要自己去校外办理
21. 学校的超市怎么样
三个超市,一些便利店
超市的物价不能说普遍便宜/贵.
我举例: "教育超市"的可乐2.7元,"南苑超市"的可乐2.9元
与此对比,一般小经销商进货可乐2.4元,销售3.0元
此处可乐均为百事可乐.
每两号教学楼会共享一个自助贩卖机,每个宿舍楼也有,只能买可乐和面包还有方便面的程度(?)
22. 学校的收发快递政策怎么样
有三个快递点.
一处为邮政快递点位于1,2宿舍附近
一处为菜鸟裹裹点位于1,2宿舍附近
一处为三号车棚点位于3,4,5,6宿舍附近
三号车棚点为京东点和顺丰点
23. 学校里面的共享单车数目与种类如何
不准
24. 现阶段学校的门禁情况如何
外出学校需要报备.各宿舍均需要刷卡开门.
25. 宿舍晚上查寝吗，封寝吗，晚归能回去吗
赛博查寝,学校安排寝室长上报人数.
是否进的去取决于宿管何时锁门.
宿管锁门是用的铁链子.
我见过他有时凌晨三点还没睡觉的所以你可以试试.

--------

至少我觉得学校对我还行,你们自己考虑.

**Comment by OPLIKATO** [comment]
补充一下：
1、微电子学院和物理学院的男生寝室基本都在12号楼，六人间，无独立卫生间。这两个学院原来是一个学院，包括电子科学、集成电路、应用物理专业，都在这个楼。其他宿舍楼都是4人间，有独卫。
2、图书馆9点半到10点之间关门
3、晚上不断电不断网
4、校园网一个月5块钱，下载速度一般在1.5MB上下。

**Reply by fautie** [reply]
嗯你们真惨。但校园网ipv6真的不限速你们可以找个隧道试一试（笑）

**Comment by deilcector** [comment]
洗澡热水能供应到11点嘛（怀疑

**Reply by ChuJiuMao** [reply]
可以，准确来说应该是澡堂12点准时停电，可以放冷水，但是热水没了

**Comment by ChuJiuMao** [comment]
补充一下原回答，今年做了很大的改革调整，以下项目发生变化：
3.大澡堂关闭了，转而每个宿舍楼有自己的公共澡堂，每个澡堂大约有6个大房间，每个大房间设有15个左右的洗浴隔间，隔间是用布帘子隔开的，就类似于
![](https://p.sda1.dev/12/a555e8318302aaafcf03fd4920289b7e/dcc61099b6346bbd.jpg)
当然，我是在12号楼住的，没有独立卫浴，因此不知道独立卫浴发生了什么变化
5.大一有早操，而且很早大约是早上6:45
8.翡翠湖校区在【小北门】安置了一个金属外卖架并配备有监控摄像，金属外卖架是没有门的，但是一般外卖流通的也比较快，【小北门】距离大部分宿舍楼都比较近
11.校园网改革，免费套餐10Mbps，具体收费规则如下图所示
![](https://p.sda1.dev/12/58942700850b99ab7533b8ee8829d1d5/Snipaste_2023-07-04_13-27-17.png)
14澡堂开放时间是12:00-24:00
15.有两个，一个是以前的大澡堂旁边有一片圈起来的场地，那里集中可以充电，然后是1、2食堂门口也有30个左右的充电桩
16.六人间限制功率1000W
22.收发快递集中于一个地方了，在1、2食堂对面
23.现在可以了，是美团的共享单车，而且很多（气煞我这个骑自己单车的人，都没地方停车了）

**Comment by fautie** [comment]
(ipv6应该还是没限速)

***@***.***

签名由 网易灵犀办公 定制


Original:
From:ChuJiuMao< ***@***.***>Date:2023-07-04 13:36:10To:CollegesChat/university-information< ***@***.***>Cc:Fautie< ***@***.***>、Author< ***@***.***>Subject:Re: [CollegesChat/university-information] 合肥工业大学（翡翠湖校区） (Discussion #33)
补充一下原回答，今年做了很大的改革调整，以下项目发生变化：
 3.大澡堂关闭了，转而每个宿舍楼有自己的公共澡堂，每个澡堂大约有6个大房间，每个大房间设有15个左右的洗浴隔间，隔间是用布帘子隔开的，就类似于

 当然，我是在12号楼住的，没有独立卫浴，因此不知道独立卫浴发生了什么变化
 5.大一有早操，而且很早大约是早上6:45
 8.翡翠湖校区在【小北门】安置了一个金属外卖架并配备有监控摄像，金属外卖架是没有门的，但是一般外卖流通的也比较快，【小北门】距离大部分宿舍楼都比较近
 11.校园网改革，免费套餐10Mbps，具体收费规则如下图所示

 14澡堂开放时间是12:00-24:00
 15.有两个，一个是以前的大澡堂旁边有一片圈起来的场地，那里集中可以充电，然后是1、2食堂门口也有30个左右的充电桩
 16.六人间限制功率1000W
 22.收发快递集中于一个地方了，在1、2食堂对面
 23.现在可以了，是美团的共享单车，而且很多（气煞我这个骑自己单车的人，都没地方停车了）

**Reply by ChuJiuMao** [reply]
细唆ipv6怎么操作_(:з」∠)_
